ddl/failtest/fail_db_test.go

```diff
@@ -334,11 +345,9 @@ func (s *testFailDBSuite) TestAddIndexWorkerNum(c *C) {
 
 	done := make(chan error, 1)
 	start := -10
-	num := 4096
 	// first add some rows
-	for i := start; i < num; i++ {
-		sql := fmt.Sprintf("insert into test_add_index values (%d, %d, %d)", i, i, i)
-		tk.MustExec(sql)
+	for i := start; i < 4090; i += 100 {
```

Comment:
Actually 4100 records are inserted, does it matter?

Reply:
`start` is -10, so I think 4090 is OK.